PyInstaller/depend/bindepend.py

```diff
@@ -711,7 +712,12 @@ def _getImports_macholib(pth):
                 lib = dyld_find(lib, executable_path=exec_path)
                 rslt.add(lib)
             except ValueError:
-                logger.error('Can not find path %s (needed by %s)', lib, pth)
+                # Starting with Big Sur, system libraries are hidden. And
+                # we do not collect system libraries on any macOS version
+                # anyway, so suppress the corresponding error messages.
+                if not in_system_path(lib):
+                    logger.error('Can not find path %s (needed by %s)',
```

Comment:
```suggestion
                    logger.error('Cannot find path %s (needed by %s)',
```

Reply:
Yeah, I was thinking about changing the message. But then, it would need to be changed in other places for consistency, and that makes it out-of-scope for this PR.

Reply:
Addressing such tiny issue when the code is changed seems reasonable. But yes, this is a very small thing to consider, it is OK for me if you change it or not :)